build_rules.gradle

```diff
@@ -802,9 +813,11 @@ ext.applyJavaNature = {
   //
   // TODO: Figure out whether we should force all dependency conflict resolution
   // to occur in the "shadow" and "shadowTest" configurations.
-  configurations.all {
-    resolutionStrategy {
-      force library.java.values()
+  configurations.all { config ->
+    if (config.getName() != "errorprone") {
```

Comment:
Its strange that `errorprone` uses a configuration, regardless a comment is warranted.

Reply:
Done